rules/windows/execution_msxsl_network.toml

```diff
@@ -23,6 +23,8 @@ timeline_id = "76e52245-7519-4251-91ab-262fb1a1728c"
 type = "eql"
 
 query = '''
+/* duplicate of MsXsl Making Network Connections - 870d1753-1078-403e-92d4-735f142edcca */
```

Comment:
The existing rule is missing `not cidrmatch(destination.ip, "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16")`

Reply:
swapped, i like the other msxsl rule better